test/test_checks.py

```diff
@@ -35,21 +36,21 @@ class DoublesOptions():
         QuoteChecker.parse_options(DoublesOptions)
 
     def test_multiline_string(self):
-        doubles_checker = QuoteChecker(None, filename=get_absolute_path('data/doubles_multiline_string.py'))
-        self.assertEqual(list(doubles_checker.get_quotes_errors(doubles_checker.get_file_contents())), [
-            {'col': 0, 'line': 1, 'message': 'Q001 Remove bad quotes from multiline string'},
+        doubles_checker = QuoteChecker(None, lines=get_lines('data/doubles_multiline_string.py'))
+        self.assertEqual(list(doubles_checker.run()), [
+            (1, 0, 'Q001 Remove bad quotes from multiline string', QuoteChecker),
```

Comment:
**Required change:** This noise is overkill and makes the test results less clear as to what actually broke. Please revert these and make a method like `get_tokens()` and pass that to the appropriate methods. There's no need to make trouble for ourselves